mne/realtime/fieldtrip_client.py

```diff
@@ -1,351 +1,351 @@
-# Author: Mainak Jas

```

Comment:
The line endings were incorrect/mixed. Now they should be consistent with the rest of the repo.